backend/src/ee/services/audit-log/audit-log-types.ts

```diff
@@ -1366,6 +1370,34 @@ interface LoadProjectKmsBackupEvent {
   metadata: Record<string, string>; // no metadata yet
 }
 
+interface CreateCertificateTemplate {
+  type: EventType.CREATE_CERTIFICATE_TEMPLATE;
+  metadata: {
```

Comment:
Add fields used when creating certificate template to the audit log metadata (mentioned elsewhere) for more complete audit trail.